conf/modules.config

```diff
@@ -291,14 +291,19 @@ process {
 // MARKDUPLICATES
 
     withName: 'SAMTOOLS_CRAMTOBAM.*'{
-        ext.args = "-b"
+        ext.args         = "-b"
+    }
+
+
+    withName: 'SAMTOOLS_STATS_CRAM'{
```

Comment:
What is happening here?

Reply:
Getting the right prefix, because the samtools/stats module now needs it.
But I'll look into making this process selector more specific, just in case.
We don't want to overwrite files

Reply:
ok, no worries, I think I might also start to mix up PRs in my head